tests/rptest/utils/cluster_topology.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2020 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from collections import defaultdict
+from rptest.services.cluster import cluster
+from rptest.services.redpanda import RedpandaService
+
+from rptest.tests.redpanda_test import RedpandaTest
+
+
+class NetemSpec():
+    def __init__(self, delay, jitter=None, loss=None):
+        self.delay = delay
+        self.jitter = 0.1 * self.delay if jitter is None else jitter
+        self.loss = loss
+
+
+class NodeQdisc():
+    def __init__(self, node, device_to_use, number_of_prio_bands):
+        self.node = node
+        self.dev = device_to_use
+        self.default_bands = 3 + number_of_prio_bands
+        self.flow = 4
+
+    def initialize(self):
+        self._tc_qdisc_add(
+            ['root', 'handle', '1:', 'prio', 'bands',
+             str(self.default_bands)])
+        self._tc_qdisc_add(['parent', '1:1', 'handle', '10:', 'sfq'])
+        self._tc_qdisc_add(['parent', '1:2', 'handle', '20:', 'sfq'])
+        self._tc_qdisc_add(['parent', '1:3', 'handle', '30:', 'sfq'])
```

Comment:
I'm curious what do these 3 qdiscs do? are they for all other traffic, then why 3?

Reply:
+1, do you mind adding some comments on how the topology maps to filters.